packages/whale-api-client/src/api/Address.ts

```diff
@@ -0,0 +1,208 @@
+import { ApiPagedResponse } from '@defichain/ocean-api-client'
+import { WhaleApiClient } from '../WhaleApiClient'
+import { LoanVaultActive, LoanVaultLiquidated } from './Loan'
+
+/**
+ * DeFi whale endpoint for address related services.
+ */
+export class Address {
+  constructor (private readonly client: WhaleApiClient) {
```

Comment:
```suggestion
  constructor (private readonly client: OceanApiClient) {
```

Can do this, since we have extra layer now

Reply:
So do this across all the APIs?

Currently this follows the conventions set in [playground](https://github.com/DeFiCh/jellyfish/blob/main/packages/playground-api-client/src/apis/Rpc.ts#L6) so wondering if we should keep in line with the established expectation or perhaps we should consolidate child clients to use `OceanApiClient` for these assignments.

Thoughts?

Reply:
IMO yes. All the API classes eventually need `this.client.requestXXXXX()` which belongs to `OceanApiClient`, unless I overlooked something.
This also circular usage like `this.client.address.xxx()`.

Reply:
should be good as 
https://github.com/DeFiCh/jellyfish/blob/d9be5e32c0a8b7310dd88f0cc110adaa93584036/packages/whale-api-client/src/WhaleApiClient.ts#L42

Reply:
@ivan-zynesis @canonbrother perhaps I can suggest that we stick with the established convention of the other non ocean client for now. Then discuss and consolidate a new approach if we see the need?